### PR TITLE
fontmanager: update to 0.8.7 + minor fixes

### PR DESCRIPTION
**Pull request by meator**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -4017,3 +4017,4 @@ libmt32emu.so.2 libmt32emu-2.5.1_1
 libqrtr-glib.so.0 libqrtr-glib-1.0.0_1
 libstemmer.so.2 libstemmer-2.1.0_1
 libdecor-0.so.0 libdecor-0.1.0_1
+libfontmanager.so.0 fontmanager-0.8.7_1
```
Comment (Chocimier) [inline common/shlibs:4020]: Remove until other package uses library, libfontmanager should be split at that time.

File: srcpkgs/fontmanager/template [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+version=0.8.7
 revision=1
 wrksrc="font-manager-${version}"
 build_helper=gir
 build_style=meson
 hostmakedepends="pkg-config yelp-tools vala-devel glib-devel gettext"
 makedepends="json-glib-devel sqlite-devel gtk+3-devel fontconfig-devel
  libsoup-devel webkit2gtk-devel"
-short_desc="Simple font management for GTK+ desktop environments"
+short_desc="Font-manager is a simple font management tool for GTK+ environments"
```
Comment (Chocimier) [inline srcpkgs/fontmanager/template:11]: Package name shouldn/t be included in desc.

**Comment by Chocimier** [comment]
Please squash.

**Comment by meator** [comment]
About the squashing, I wasn't sure how should I format the commit message, since these changes don't fit in any of categories listed in `CONTRIBUTING.md`, so I made multiple commits. I'll fix this soon.

**Comment by Chocimier** [comment]
You can use message body for things over simple update.

File: srcpkgs/fontmanager/template (hunk 2 above)
Comment (meator) [reply srcpkgs/fontmanager/template:11]: I do this to make searching for this package simpler. There is a little unfortunate inconsistency between the Voidlinux package name and the actual package name (Voidlinux uses `fontmnager` and practically every other package manager uses the official name `font-manager`). This change change costs basically nothing and it makes font-manager searchable via `xbpq-query -Rs font-manager`. If you do not agree with me, I will remove this.

Comment (Chocimier) [reply srcpkgs/fontmanager/template:11]: Uh, okay.

**Pull request merged**